src/Components/Home.js

```diff
@@ -1,12 +1,17 @@
 import './Home.css'
-
```

Comment:
Is there a reason for removing this line?

Reply:
@krishguptadev No, There is no reason as such, But if you want I can add it again.

Reply:
I think that it helps separate the imports and it looks pleasing to the eye to